src/Utils.ts

```diff
@@ -61,6 +62,40 @@ export function printCurrentVersion() {
 	}
 }
 
+export function ensureTypescriptInstalled(): never | void {
+	const NO_TYPESCRIPT_MESSAGE = `
+  FuseBox failed to initialize. Please check that:
+      - you have TypeScript installed
+          -> if using NPM, run on your terminal:\n
+              npm i -D typescript\n
+          -> if using Yarn, run on your terminal:\n
+              yarn add typescript --dev\n
+      - the TypeScript version installed is >= ${MINIMAL_TYPESCRIPT_VERSION}
+`;
+	const TYPESCRIPT_NOT_SUPPORTED = `
+  FuseBox failed to initialize. Please check that:
+      - the TypeScript version installed is >= ${MINIMAL_TYPESCRIPT_VERSION}
+`;
+	try {
+		const ts = require("typescript");
+		if (typeof ts.versionMajorMinor === "string") {
+			if (Number(MINIMAL_TYPESCRIPT_VERSION) > Number(ts.versionMajorMinor)) {
+				console.log(TYPESCRIPT_NOT_SUPPORTED);
+				process.exit(1);
+			}
+			return
+		}
+		console.log(NO_TYPESCRIPT_MESSAGE);
+		process.exit(1);
+	} catch (error) {
+		if (error.code === "MODULE_NOT_FOUND") {
+			console.log(NO_TYPESCRIPT_MESSAGE);
+			process.exit(1);
+		}
+		throw error;
```

Comment:
Do you think we can make it more beautiful ?
<img width="909" alt="screenshot 2019-01-28 20 56 49" src="https://user-images.githubusercontent.com/1629088/51862821-bfd79800-233f-11e9-880d-e3991bc78bc6.png">

I think we can just take the first line from the stack trace and print it nicely. like we print that red message